lib: location: Remove HERE services (#20372)
nrfconnect/sdk-nrf: doc/nrf/libraries/modem/location.rst

  * Services can be handled by the application by enabling the :kconfig:option:`CONFIG_LOCATION_SERVICE_EXTERNAL` Kconfig option, in which case rest of the service configurations are ignored.
  * The service is selected in the :c:struct:`location_method_config` structure when requesting for location.

>   * The services available is `nRF Cloud Location Services <nRF Cloud Location Services documentation_>`_.
>   * The data transport method for the `nRF Cloud Location Services <nRF Cloud Location Services documentation_>`_ can be configured to either MQTT (:kconfig:option:`CONFIG_NRF_CLOUD_MQTT`) or REST (:kconfig:option:`CONFIG_NRF_CLOUD_REST`).

Reviewer: ```suggestion
    * You can configure the data transport method for the `nRF Cloud Location Services <nRF Cloud Location Services documentation_>`_ to either MQTT (:kconfig:option:`CONFIG_NRF_CLOUD_MQTT`) or REST (:kconfig:option:`CONFIG_NRF_CLOUD_REST`).
```
I believe we don't need the line I'm suggesting to remove.

Location service accounts
=========================

> To use the location services that provide A-GNSS or P-GPS, cellular or Wi-Fi positioning data, see `nRF Cloud Location Services <nRF Cloud Location Services documentation_>`_ documentation for setting up your account and getting the required credentials for authentication.

Reviewer: ```suggestion
To use the location services that provide A-GNSS, P-GPS, cellular, or Wi-Fi positioning data, see the `nRF Cloud Location Services <nRF Cloud Location Services documentation_>`_ documentation for setting up your account and getting the required credentials for authentication.
```
